src/reactor.rs

```diff
@@ -114,7 +116,15 @@ impl Reactor {
                 let stderr_output = stderr_buffer.lock().unwrap().clone();
 
                 let report = self.report_builder.identify(exit_status.success(), &stdout_output, &stderr_output);
-                self.config.notifier.notify(report);
+                let icon = match report.outcome {
+                    Outcome::TestsPassed => "face-angel",
+                    Outcome::TestsFailed | Outcome::CompileError => "face-angry"
+                };
+                Notification::new()
+                    .summary(report.title())
```

Comment:
Looks like we need to set `body` here as well, when `report.detail` is `Some(String)`